package.json

```diff
@@ -142,19 +142,18 @@
     "gulp-coffee": "^2.2.0",
     "gulp-inline-source": "^2.1.0",
     "gulp-shell": "^0.5.2",
-    "husky": "^4.2.3",
+    "husky": "^4.2.5",
     "intercept-stdout": "^0.1.2",
-    "mocha": "^6.2.2",
+    "mocha": "^6.2.3",
     "mock-require": "^3.0.3",
     "nock": "^11.9.1",
     "parse-link-header": "~1.0.1",
     "pkg": "^4.4.2",
-    "prettier": "^1.19.1",
```

Comment:
This is unnecessary as the prettifying is handled by balena-lint now (which installs its own version of prettier, and this one could potentially clash)